src/include/parser/pg_trigger.h

```diff
@@ -0,0 +1,81 @@
+#ifndef PG_TRIGGER_H
+#define PG_TRIGGER_H
+
+/* ----------------
+ *		compiler constants for pg_trigger
+ * ----------------
+ */
+#define Natts_pg_trigger 15
```

Comment:
This file is copied from Postgres parser. We need to keep the macros the same to interpret the parsing tree generated by the Postgres parser.